cameo/api/adapter.py

```diff
@@ -0,0 +1,150 @@
+import re
+from cameo.data import metanetx
+from cameo.core.reaction import Reaction
+from cameo.core.metabolite import Metabolite
+import logging
+logger = logging.getLogger(__name__)
+
+
+def clean_bigg_id(string):
+    return re.sub(r"bigg:|dsh", "", string)
+
+
+def get_existing_metabolite(mnx_id, model, compartment):
+    """Find compartment in the model by Metanetx id.
+
+    Parameters
+    ----------
+    mnx_id - Metanetx id
+    model - cobra model
+    compartment - f.e "_c"
+
+    Returns
+    -------
+    Metabolite or None
+
+    """
+    if not mnx_id:
+        return
+    try:
+        clean_id = clean_bigg_id(metanetx.mnx2bigg[mnx_id])
+        return model.metabolites.get_by_id(clean_id + compartment)
+    except KeyError:
+        try:
+            return model.metabolites.get_by_id(mnx_id + compartment)
+        except KeyError:
+            pass
+
+
+def contains_carbon(metabolite):  # TODO: use method from Metabolite class when this change is merged
+    if not metabolite.formula:
+        raise ValueError("No formula for metabolite {}, it's unknown if there is carbon in it")
+    return 'C' in metabolite.elements
+
+
+def find_metabolite_info(met_id):
+    """Find chemical formula of metabolite in metanetx.chem_prop dictionary
+
+    :param met_id: string of format "<metabolite_id>_<compartment_id>", where <metabolite_id> is a BIGG id or a
```

Comment:
i guess better use google (?) style docstrings like rest of cameo (i just updated a pr to adhere to that myself (:)